seacrowd/sea_datasets/brcc/brcc.py

```diff
@@ -0,0 +1,111 @@
+  title     = {BRCC and SentiBahasaRojak: The First Bahasa Rojak Corpus for Pretraining and Sentiment Analysis Dataset},
+  booktitle = {Proceedings of the 29th International Conference on Computational Linguistics},
+  publisher = {International Committee on Computational Linguistics},
+  year      = {2022},
+  url       = {https://aclanthology.org/2022.coling-1.389/},
+  pages     = {4418--4428},
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["zlm", "eng", "cmn"]
+_DATASETNAME = "brcc"
+_DESCRIPTION = """
+The Bahasa Rojak Crawled Corpus (BRCC) is a code-mixed dataset for the Bahasa Rojak dialect in Malaysia.
+Passages are generated through data augmentation from English and Malay Wikipedia pages using a modified CoSDA-ML method.
+The quality of generated passages is evaluated by two native Malay speakers.
+"""
+_HOMEPAGE = "https://data.depositar.io/dataset/brcc_and_sentibahasarojak"
+_LICENSE = Licenses.UNKNOWN.value
+_URL = "https://data.depositar.io/dataset/304d1572-27d6-4549-8292-b1c8f5e9c086/resource/8a558f64-98ff-4922-a751-0ce2ce8447bd/download/BahasaRojak_Datasets.zip"
+
+_SUPPORTED_TASKS = [Tasks.SELF_SUPERVISED_PRETRAINING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class BRCCDataset(datasets.GeneratorBasedBuilder):
+    """Dataset of Bahasa Rojak passages generated from English and Malay Wikipedia pages."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_ssp",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd ssp schema",
+            schema="seacrowd_ssp",
+            subset_id=_DATASETNAME,
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        # Source schema = SeaCrowd schema because file only contains lines of text
+        if self.config.schema in ("source", "seacrowd_ssp"):
+            features = schemas.ssp_features
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        data_dir = dl_manager.download_and_extract(_URL)
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "BahasaRojak Datasets/BRCC/mix.train"),
```

Comment:
Just a tiiiiny bit. I think this is better if we use `os.path.join(data_dir, "BahasaRojak Datasets", "BRCC", "mix.train")` as different OS might have different slash. Sometimes I do get problem with slashing, although this script works with my Windows.

![image](https://github.com/SEACrowd/seacrowd-datahub/assets/71018093/2eddcbff-8dcc-4646-bea6-9452adb67592)